c2corg_ui/templates/profile/data.html

```diff
@@ -0,0 +1,80 @@
+<%!
+from c2corg_ui.templates.utils import get_lang_lists
+from json import dumps
+%>
+<%namespace file="helpers/detailed_profile_attributes.html" import="get_profile_general,
+    show_profile_float_buttons"/>
+<%namespace file="../helpers/view.html" import="get_comments, get_image_gallery,
+    photoswipe_gallery, show_attr, show_missing_langs_links, show_other_langs_links,
+    show_badge, get_document_description"/>
+
+% if not_authorized:
+  <section class="view-details-section">
+    <div class="view-details-description col-xs-12 description">
+      <span class="lead">
+        <div id="document-description" class="collapse in" translate>This profile is only available to authenticated users.</div>
+      </span>
+    </div>
+  </section>
+
+% else:
+  <%
+      user_id = profile['document_id']
+      # FIXME: not needed!?
+      other_langs, missing_langs = get_lang_lists(profile, lang)
+  %>
+ </header>
```

Comment:
Definitely not. Probably a missed line while reorganising the HTML. I will fix this, thanks.